commands/info.md

```diff
@@ -73,6 +73,9 @@ Here is the meaning of all fields in the **clients** section:
 
 *   `connected_clients`: Number of client connections (excluding connections
      from replicas)
+*   `cluster_connections`: An approximation of the number of sockets used by the
+     cluster's bus
+*   `maxclients`: The value of the `maxclients` configuration directive
```

Comment:
do we wanna mention that it should be matched with the sum of connected_clients, connected_slaves, and cluster_connections (to be able to tell how much remain available)?

Reply:
That's a good idea unless perhaps we want to add a new `remaining_connections` field that reflects this metric?

Reply:
it was suggested, but IIRC we decided to just reflect raw metrics and let the user do the math.

Reply:
Please review the addition made in 6a754bb